setup.py

```diff
@@ -108,7 +108,7 @@ def build_extension(self, ext):
         author="Frida Developers",
         author_email="oleavr@frida.re",
         url="https://frida.re",
-        install_requires=["setuptools", "typing_extensions; python_version<'3.8'"],
+        install_requires=["setuptools", "typing_extensions; python_version<'3.11'"],
```

Comment:
This is not really related to this MR, but setuptools should not be defined as a dependency, because it is only needed during build time.

Instead it should be specified in the `pyproject.toml` according [PEP 517](https://peps.python.org/pep-0517/) and [PEP 518](https://peps.python.org/pep-0518/)

i.e.  :
```toml
[build-system]
requires = ["setuptools"]
build-backend = "setuptools.build_meta"
```

Reply:
Would keep in mind but fix in a separate PR